src/store/chat/bus.ts

```diff
@@ -0,0 +1,49 @@
+import { eventChannel, multicastChannel } from 'redux-saga';
+import { call } from 'redux-saga/effects';
+import { chat } from '../../lib/chat';
+
+export enum Events {
+  MessageReceived = 'chat/message/received',
+  MessageDeleted = 'chat/message/deleted',
+  UnreadCountChanged = 'chat/message/unreadCountChanged',
+  ReconnectStart = 'chat/recconectStart',
+  ReconnectStop = 'chat/recconectStop',
+  InvalidToken = 'chat/invalidToken',
+}
+
+let theBus;
+export function* getChatBus() {
```

Comment:
After some discussion with Ratik we thought we'd migrate the names for redux-saga channels to "bus" to avoid confusion with our domain "channels". Tried it here and it seems to feel ok.